admin/admin-base.php

```diff
@@ -187,11 +187,15 @@ public function admin_enqueue_scripts() {
 			}
 
 			// Block editor with legacy metabox in WP 5.0+.
-			if ( method_exists( $screen, 'is_block_editor' ) && $screen->is_block_editor() && ! pll_use_block_editor_plugin() ) {
+			if ( $this->is_block_editor( $screen ) ) {
 				$scripts['block-editor'] = array( array( 'post' ), array( 'jquery', 'wp-ajax-response', 'wp-api-fetch', 'jquery-ui-dialog', 'wp-i18n' ), 0, 1 );
 			}
```

Comment:
Not sure since we check for translatable post type [here](https://github.com/polylang/polylang/blob/3.2.5/admin/admin-base.php#L181) before adding the script for the block editor.

Reply:
If I simplify your code, it looks like this:
```
		if ( ! empty( $screen->post_type ) && $this->model->is_translated_post_type( $screen->post_type ) ) {
			// Block editor with legacy metabox in WP 5.0+.
			if ( $this->is_block_editor( $screen ) ) {
				$scripts['block-editor'] = array( array( 'post' ), array( 'jquery', 'wp-ajax-response', 'wp-api-fetch', 'jquery-ui-dialog', 'wp-i18n' ), 0, 1 );
			}
		}

		if ( $this->is_block_editor( $screen ) ) {
			$scripts['block-editor'] = array( array( 'widgets', 'site-editor' ), array( 'jquery', 'wp-ajax-response', 'wp-api-fetch', 'jquery-ui-dialog', 'wp-i18n' ), 0, 1 );
		}
```
The the block editor script is added 2 times:
1. when a translated post type is edited
2. inconditionnally

When 2 is added, 1 is added too. So it looks useless to me.